### ISSUE TITLE
Binding mouse buttons to swap tool doesn't work

### ISSUE THREAD

**Issue by Hnaguski** [gigaherz/ToolBelt#65]
I've noticed that on my machine when I try to bind the swap tool button to mouse 4 or 5 (the side buttons on my mouse) the button doesn't work. Additionally the game log fills with the following:
```
[Render thread/ERROR] [minecraft/MainWindow]: ########## GL ERROR ##########
[Render thread/ERROR] [minecraft/MainWindow]: @ Pre render
[Render thread/ERROR] [minecraft/MainWindow]: 65539: Invalid key 3
```
Which just keeps printing as long as the game is running. Other keys work fine and don't spam the log.

I am running Debian Linux on an AMD gpu if that helps at all.

**Comment by rodrigonramon** [comment]
2nd this, tried and it didnt work.

**Comment by gigaherz** [comment]
Hmm I must have missed this issue report somehow, I'll look into it as soon as I can.

**Comment by gigaherz** [comment]
Hmm I think I already fixed this in the previous release but in any case, it should be fixed in v1.16.0.
